### PR TITLE
Feature/item taxonomy display

### PR DESCRIPTION
**Pull request by akisys**
Allows the display of tags assigned to each Post. Let me know what you think!

### CODE REVIEW

**Comment by ParitoshBh** [comment]
Note that the theme doesn't support displaying articles under a tag. Once this PR is merged, I'll add that functionality.

**Comment by ParitoshBh** [comment]
Also, please update README accordingly.

**Comment by akisys** [comment]
> Also, please update README accordingly.

Good point, and also done for this little switch. Let me know if it's sufficient.

**Comment by ParitoshBh** [comment]
@akisys Did you have a chance to review my comment above?

**Comment by akisys** [comment]
Hey @ParitoshBh 
It's been a while since I looked at the PR code for this. Afaik I'm not using Pages but only Posts with this Taxonomy feature.

Can you please elaborate on your open points with this?!

File: templates/blog.html.twig [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% extends 'partials/base.html.twig' %}
 
 {% if uri.param('category') %}
-	{% set page_title = "Category: " ~ uri.param('category') %}
+{% elseif uri.param('tag') %}
```
[the remark this answers is not in the document]
Comment (akisys) [reply templates/blog.html.twig:4]: I will look into this this weekend!

[the remark this answers is not in the document]
Comment (akisys) [reply templates/blog.html.twig:4]: This is covered on PR #20 , there was a slight mix-up in the order of PRs.

#20 adds the override of Labels and default back to to what you are asking for.

**Pull request merged**